tests/subsys/sd/mmc/testcase.yaml

```diff
@@ -0,0 +1,13 @@
+common:
+  depends_on: sdhc
+  tags: drivers sdhc
+tests:
+  subsys.sd.mmc:
+    harness: ztest
+    harness_config:
+      fixture: fixture_sdhc
+    filter: dt_alias_exists("sdhc0")
```

Comment:
I think we may actually want to change the filter here. This test targets eMMC, which is a soldered IC and will always be present on a board (if it is available). In contrast, SDMMC cards can be removed from the board (hence why `fixture` was used).

Given this, maybe we should depend on an MMC node label or alias being present. If we did this, we could remove the `fixture` tag, as there would be no specific fixture required to run this test case.

Reply:
Updated